src/parseChangelog.test.js

```diff
@@ -196,6 +196,119 @@ describe('parseChangelog', () => {
     ]);
   });
 
+  it('should parse changelog with multi-line change description', () => {
+    const changelog = parseChangelog({
+      changelogContent: outdent`
+        # Changelog
+        All notable changes to this project will be documented in this file.
+
+        The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+        and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+        ## [Unreleased]
+
+        ## [1.0.0] - 2020-01-01
+        ### Changed
+        - Something else
+        Further explanation of changes
```

Comment:
is a multiline of text valid?

ex:

```
        ## [1.0.0] - 2020-01-01
        ### Changed
        Hello World
        Further explanation of changes
```

Reply:
Yep! Any number of lines will be accepted as a valid change description.